Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/NetworkConnectionldap_log4j.yaml

```diff
@@ -1,6 +1,8 @@
 id: 19abc034-139e-4e64-a05d-cb07ce8b003b
 name: Malicious Connection to LDAP port for CVE-2021-44228 vulnerability
 description: |
+   'This query detects exploitation attempts for CVE-2021-44228 involving log4j vulnerability by looking for connections to default LDAP ports. Log4j is an open-source Apache logging library used in many Java-based applications'
```

Comment:
This is enough ' This query detects exploitation attempts for CVE-2021-44228 involving log4j vulnerability by looking for connections to default LDAP ports. '

Reply:
Resolved